temporalcli/commands.operator_namespace.go

```diff
@@ -336,6 +337,7 @@ func printNamespaceDescriptions(cctx *CommandContext, responses ...*workflowserv
 				"ReplicationConfig.Clusters", "Config.HistoryArchivalState", "Config.VisibilityArchivalState",
 				"IsGlobalNamespace", "FailoverVersion", "FailoverHistory", "Config.HistoryArchivalUri",
 				"Config.VisibilityArchivalUri",
+				"Config.CustomSearchAttributeAliases",
```

Comment:
this will show CustomSearchAttributeAliases by default. Can add a flag to display as an option as well.